app/js/index.js

```diff
@@ -18,6 +21,15 @@
 	var validSlackSubdomain = /(.+)\.slack.com/i;
 	var slackDownloadHostname = 'files.slack.com';
 
+	// Interpet our CLI arguments
+	// DEV: We need to coerce `nw.js'` arguments as `commander` expects normal (`['node', 'slack-for-linux', '--help']`)
+	//   but `nw.js` only provides arguments themselves (e.g. `--help`)
+	var argv = ['nw', pkg.name].concat(gui.App.argv);
+	program
+		.version(pkg.version)
+		.option('--minimize-to-tray')
```

Comment:
Can we add a description for the option? It is what's seen when someone runs `--help`

```js
.option('--minimize-to-tray', 'Hide/show window to tray rather than minimize/restore')
```